### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

**Comment by XVilka** [comment]
Why removing lgtm config?

**Comment by xarkes** [comment]
Because it was not working.
I find it stupid to ask LGTM to analyze r2 submodule while it's already being analyzed in its main repository but I couldn't find a way to ignore it.

**Comment by Maijin** [comment]
@sjvs may have an idea on how to do the ignore ❓

File: src/core/MainWindow.cpp [hunk 1]
```diff
@@ -429,7 +429,7 @@ void MainWindow::addMenuFileAction(QAction *action)
     ui->menuFile->addAction(action);
 }
 
-void MainWindow::openNewFile(InitialOptions options, bool skipOptionsDialog)
+void MainWindow::openNewFile(InitialOptions& options, bool skipOptionsDialog)
```
Comment (thestr4ng3r) [inline src/core/MainWindow.cpp]:
```suggestion
void MainWindow::openNewFile(InitialOptions &options, bool skipOptionsDialog)
```

File: src/core/MainWindow.h [hunk 2]
```diff
@@ -65,7 +65,7 @@ class MainWindow : public QMainWindow
     explicit MainWindow(QWidget *parent = nullptr);
     ~MainWindow() override;
 
-    void openNewFile(InitialOptions options = InitialOptions(), bool skipOptionsDialog = false);
+    void openNewFile(InitialOptions& options, bool skipOptionsDialog = false);
```
Comment (thestr4ng3r) [inline src/core/MainWindow.h]:
```suggestion
    void openNewFile(InitialOptions &options, bool skipOptionsDialog = false);
```

**Comment by sjvs** [comment]
> @sjvs may have an idea on how to do the ignore

I'm sure @imsolost and/or @samlanning can chip in!

**Comment by samlanning** [comment]
Yep, I think this configuration should work:

```
path_classifiers:
  library:
    - radare2
```

The current configuration excludes `radare2` from the classification, which means that it's results get *included* as it's considered *not* generated code. the `excluded:` modifier in the configuration is primarily for overriding LGTM's default classification, or for previous rules specified in the same config.

**Comment by xarkes** [comment]
Thanks, let me try it! :)

**Pull request merged**